### DISCUSSION TITLE
Plus de paramètres dans l’éditeur de texte

### DISCUSSION THREAD

**Discussion by Florian57-DI** [isontheline/pro.webssh.net#853] [Ideas]
Bonjour
Serait il possible d'avoir plus de paramètres dans l'éditeur de texte? La taille de la police ? La couleur etc 
Merci bcp

**Comment by isontheline** [comment]
Bonjour @Florian57-DI 👋

Au niveau de l'éditeur de texte, parle-t-on bien du SFTP ?
Oui je peux ajouter des options pour changer la taille de la police.

Au niveau de la "couleur" j'aurais besoin de plus de précisions. 
Actuellement la coloration syntaxique s'adapte en fonction du type de fichier. 
Rencontrez-vous des problèmes à ce niveau ?

Bonne soirée à vous

**Reply by Florian57-DI** [reply]
Merci de votre réponse très rapide !! 

Oui je parle bien du SFTP. La couleur, non ce n'est pas une obligation. C'était plutôt de choisir un fond foncé par exemple (comme le thème monokai par exemple).

Je m'en sert quasiment tous les jours, je suis développeur. Mais c'est surtout l'écriture qui est un peu trop grosse et c'est assez embêtant ! 

Sinon excellente appli ! La meilleure pour moi, et je les ai quasiment toutes testées !

Merci d'avance !

**Reply by isontheline** [reply]
Merci infiniment pour vos compliments 🙏

Entendu je vais ajouter les deux fonctionnalités suivantes :
* Possibilité de modifier la taille de la police : https://github.com/isontheline/pro.webssh.net/issues/849
* Possibilité de choisir un thème différent pour l'éditeur : https://github.com/isontheline/pro.webssh.net/issues/854

Je n'ai pas encore de date planifiée mais je vais faire mon possible pour les traiter "rapidement".

A nouveau merci !

**Reply by Florian57-DI** [reply]
Ah c'est génial ! 
Merci bcp !!!!

Bonne soirée, 
Florian

**Reply by isontheline** [reply]
Dernière question Florian : quelles plateformes utilisez-vous prioritairement ?
* iPad
* iPhone
* Mac

Sur Mac la police est déjà personnalisable via le menu Présentation > Zoom avant / arrière, sur iPad et iPhone je vais le rajouter.

**Reply by Florian57-DI** [reply]
Je l'utilise bcp sur iPad et iPhone 😅

Merci a vous

**Reply by isontheline** [reply]
Bonjour Florian @Florian57-DI 👋

La dernière beta doit vous permettre de modifier la taille par défaut de l'éditeur de texte SFTP : 
https://testflight.apple.com/join/QSrBK59z

iOS Settings > WebSSH > SFTP > Text Editor > Default Font Size

Pourriez-vous y jeter un oeil ?

**Reply by Florian57-DI** [reply]
Bonsoir,

Mais quelle rapidité !

C'est exactement ça ! Génial !
Merci bcp

Florian
